.github/workflows/test-template.yml

```diff
@@ -1061,10 +1027,9 @@ jobs:
       matrix:
         os: [macos-latest, windows-latest]
         node: [14]
-        queryEngine: ${{ fromJson(needs.determineBuildMatrixParams.outputs.queryEngine) }}
+        queryEngine: ${{ fromJson(needs.inputs.queryEngine) }}
```

Comment:
shoud this be
```suggestion
        queryEngine: ${{ fromJson(inputs.queryEngine) }}
```
?

Reply:
Good catch, why didn't github yell at me?